src/App.tsx

```diff
@@ -1,16 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const title = movie.title.toLowerCase();
+    const description = movie.description.toLowerCase();
+    const filteredQuery = query.toLowerCase();
```

Comment:
we can skip those variables and create only booleans for checking a condition.
Let's say `doesTitleMatch / doesDescriptionMatch`